operator/v1/0000_70_cluster-network-operator_01_crd.yaml

```diff
@@ -239,6 +239,7 @@ spec:
                         policyAuditConfig:
                           description: policyAuditConfig is the configuration for network policy audit events. If unset, reported defaults are used.
                           type: object
+                          default: null
```

Comment:
this looks wrong. What is kubebuilder doing here?

Reply:
Not sure Maybe I need to specify the object here? Let me give that a try.

Reply:
Two ways to solve this:
1. fix kubebuilder's controller-gen upstream
2. use a yaml patch as we do in similar cases where kubebuilder is broken. Look in this repo for yaml-patch files as blueprint. A yaml patch allows you to set `{}` here explicitly as value. It is applied **after** controller-gen has been run.

Reply:
+1 sounds good I'll give a stab at debugging kubebuilder first, thanks!